### PR TITLE
fix: install just autocomplete

### PR DESCRIPTION
**Pull request by madwort**
* fixes https://github.com/opensafely-core/backend-server/issues/158
* manually installed on `emis` to verify it works as expected

### CODE REVIEW

File: scripts/bootstrap.sh [hunk 1]
```diff
@@ -31,6 +31,7 @@ if [[ "$OSTYPE" == "linux-gnu"* ]]
 then
   echo "Installing just in /usr/local/bin/just"
   install -m 755 "$PWD/bin/just-1.14.0-x86_64-unknown-linux-musl" /usr/local/bin/just
+  just --completions bash > /etc/bash_completion.d/just
```
Comment (bloodearnest) [inline scripts/bootstrap.sh:34]: Will this take effect immediately in the shell I ran bootstrap from? Or does in need a new shell to activate?

**Review by bloodearnest — APPROVED** [review]
approved with a question

File: scripts/bootstrap.sh (hunk 1 above)
Comment (madwort) [reply scripts/bootstrap.sh:34]: pretty sure it needs a new shell, same as changes to profile

Comment (madwort) [reply scripts/bootstrap.sh:34]: should we update docs for install procedure?

Comment (madwort) [reply scripts/bootstrap.sh:34]: updated

**Pull request merged**